.golangci.yaml

```diff
@@ -0,0 +1,64 @@
+run:
+  # define go version
+  go: "1.23"
+
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+
+  # include test files or not, default is true
+  tests: true
+
+# output configuration options
+output:
+  formats:
+    # colored-line-number|line-number|json|tab|checkstyle, default is "colored-line-number"
+    - format: colored-line-number
+
+  # print lines of code with issue, default is true
+  print-issued-lines: true
+
+  # print linter name in the end of issue text, default is true
+  print-linter-name: true
+
+linters-settings:
+  depguard:
+    rules:
+      Main:
+        deny:
+          - desc: "Use github.com/go-kit/log instead of github.com/go-kit/kit/log"
+            pkg: github.com/go-kit/kit/log
+
+linters:
+  enable:
+    - copyloopvar
+    - depguard
+    - errcheck
+    - gci
+    - goconst
+    - gofmt
+    - gochecksumtype
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - revive
+    - staticcheck
+    - typecheck
+    - unconvert
```

Comment:
Hi might you add [testifylint](https://golangci-lint.run/usage/linters/#testifylint) as well